server/re/src/reDataObjOpr.cpp

```diff
@@ -993,11 +993,11 @@ msiDataObjRepl( msParam_t *inpParam1, msParam_t *msKeyValStr,
         return rei->status;
     }
 
-    validKwFlags = OBJ_PATH_FLAG | DEST_RESC_NAME_FLAG | NUM_THREADS_FLAG |
-                   BACKUP_RESC_NAME_FLAG | RESC_NAME_FLAG | UPDATE_REPL_FLAG |
-                   REPL_NUM_FLAG | ALL_FLAG | ADMIN_FLAG | VERIFY_CHKSUM_FLAG |
-                   RBUDP_TRANSFER_FLAG | RBUDP_SEND_RATE_FLAG | RBUDP_PACK_SIZE_FLAG |
-                   FORCE_CHKSUM_FLAG;
+    validKwFlags = OBJ_PATH_FLAG | DEST_RESC_NAME_FLAG | NUM_THREADS_FLAG | RESC_NAME_FLAG | UPDATE_REPL_FLAG |
+                   REPL_NUM_FLAG | ALL_FLAG | ADMIN_FLAG | VERIFY_CHKSUM_FLAG | RBUDP_TRANSFER_FLAG |
```

Comment:
ooh the RBUDP flag too... hmm #6610

Reply:
Yep, that will need to be marked for deprecation I suppose.